src/pages/index.js

```diff
@@ -138,17 +192,30 @@ const validationSettings = {
 
 const editFormElement = editProfileModal.querySelector(".modal__form");
 const addCardElement = addCardModal.querySelector(".modal__form");
```

Comment:
You have already found the 4 forms previously in lines 62, 64, 68, 70

Please, use them in the code